src/ProjectOrigin.RequestProcessor/Services/Projector.cs

```diff
@@ -0,0 +1,40 @@
+using System.Reflection;
+using ProjectOrigin.RequestProcessor.Interfaces;
+
+namespace ProjectOrigin.RequestProcessor.Services;
+
+public class Projector : IProjector
+{
+    private Type type;
+    private Dictionary<Type, MethodInfo> applyDict;
+
+    public Projector(Type type)
+    {
+        this.type = type;
+
+        var methonName = nameof(IModelProjectable<object>.Apply);
```

Comment:
methonName - looks like a typo in methodName. However it is consistent and it works :-)